### ISSUE TITLE
"uslugify" breaks header anchors list

### ISSUE THREAD

**Issue by CMTV** [squidfunk/mkdocs-material#972]
## Description

Setting custom slugify function "uslugify" from pymodown breaks header anchors block at the right.

```yml
markdown_extensions:
    ...
    - toc:
          permalink: '#'
          slugify: !!python/name:pymdownx.slugs.uslugify
    ...
```

### Expected behavior

When scrolling down headers are fading out. This gives a visual understanding of current position on the page.

### Actual behavior

When scrolling, the browser console spams the following error:
![error](https://user-images.githubusercontent.com/25921286/51799426-d98ab980-2231-11e9-8bb9-02590ed9dd64.png)

![error2](https://user-images.githubusercontent.com/25921286/51799494-266e9000-2232-11e9-9334-7ad571773735.png)

### Steps to reproduce the bug

1. In `mkdocs.yml` set a custom slugify function ("uslugify" in my case)
2. Add some cyrillic headers in `index.md`
3. Anchor links are not fading and the console spams the error when scrolling the page

**Comment by squidfunk** [comment]
Thanks for reporting, I can confirm this issue. Will look into it.

**Comment by squidfunk** [comment]
I tried to reproduce your test case and think I have a potential fix in #ee0371d. Please try the master to see if the issue persists.

**Comment by CMTV** [comment]
Pardon, how can I update mkdocs theme?

**Comment by CMTV** [comment]
Ok. I can confirm it is fixed when directly adding `material` folder and setting `custom_dir: 'material'`.

**Comment by squidfunk** [comment]
The idea was that you check out the latest source from GitHub ([see instructions](https://squidfunk.github.io/mkdocs-material/getting-started/#cloning-from-github)) and validate whether the error is gone. Did you do that?

**Comment by CMTV** [comment]
Yep, I did. The bug is fixed.

**Comment by squidfunk** [comment]
Perfect! I will issue a release within the next few days. Until then, please use the current `master`.

**Comment by squidfunk** [comment]
Fix released as part of 3.3.0.
